post_installation_configuration/coreos-layering.adoc

```diff
@@ -17,7 +17,7 @@ With {op-system} image layering, you can install RPMs into your base image, and
 
 [IMPORTANT]
 ====
-RPMs installed through a custom layered image can conflict with RPMs installed by using a machine config. It is recommended to use either machine config or a custom layered image to add extensions, but not both, unless you are certain there will be no conflicts. If there is a conflict, the MCO enters a `degraded` state when it tries to install the machine config RPM. You need to remove the conflicting extension from your machine config before proceeding.
+It is recommended to not apply realtime kernel and extensions RPMs as custom layered content. RPMs installed through a custom layered image can conflict with RPMs installed by using a machine config. If there is a conflict, the MCO enters a `degraded` state when it tries to install the machine config RPM. You need to remove the conflicting extension from your machine config before proceeding.
```

Comment:
> It is recommended to not apply

I think that this phrasing is hard to follow, and some of the existing text in this IMPORTANT block could be clarified as well. At minimum, please change `It is recommended to not apply` to `It is not recommended to apply` or similar. 

Here's a suggested rewrite for you to consider:

Suggestion:
```suggestion
Installing realtime kernel and extensions RPMs as custom layered content is not recommended. This is because these RPMs can conflict with RPMs installed by using a machine config.

If there is a conflict, the MCO enters a `degraded` state when it tries to install the machine config RPM. To resolve this, remove the conflicting extension from the machine config before proceeding.
```